scripts/globals/magic.lua

```diff
@@ -1027,8 +1039,6 @@ end;
 
 function getHelixDuration(caster)
     --Dark Arts will further increase Helix duration, but testing is ongoing.
-
-    local casterLevel = caster:getMainLvl();
```

Comment:
You can't remove this variable. It's in a separate function scope. If you cast a helix spell with this code as it is, your server will crash.

Reply:
Woops, will correct.